README.md

```diff
@@ -1 +1,19 @@
-# git_process_demo
\ No newline at end of file
+# git_process_demo
+
+## Introduction
+This is the introduction for the project.
+
+### Purpose
+There are two purposes of this project:
+1. To document examples of pull requests.
+2. That's it.
+
+### Scope
+Beyond providing an example of good documentation practices, there is no other usage for this specific repository. 
+However, the code contained herein is an excellent demonstration of a Dotnet project, that can be downloaded by the public. It showcases some of the features and power within the C# programming language and can be consulted for a great idea on how to develop for Windows desktop and devices.
+
+## Git the code
+Using a combination of GitHub and SourceTree, contributing to and tracking changes for this project is easier than ever!
+1. Make an account on GitHub.com, and contact the repository owner for an invite to be a Contributor.
+2. Use SourceTree to clone this repository to your local machine. The .git link can be found at the top of the repository's home page on GitHub.
+3. Use your favourite IDE to make changes to the code.
```

Comment:
Why did you use the British spelling of favorite?

Reply:
It's an alternative spelling.

Reply:
Okay, that's fine. However, in line 18, please change the text from 
'The .git link' to 'Our repository's  web URL'
Thanks.

Reply:
Will do.